API/NoDocs/Microsoft.Extensions.Diagnostics.ResourceMonitoring/LinuxUtilizationExtensions.cs

```diff
@@ -0,0 +1,13 @@
+// Assembly 'Microsoft.Extensions.Diagnostics.ResourceMonitoring'
+
+using System;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Extensions.Diagnostics.ResourceMonitoring;
+
+public static class LinuxUtilizationExtensions
+{
+    public static IResourceMonitorBuilder AddLinuxProvider(this IResourceMonitorBuilder builder);
```

Comment:
I thought we were going to mrege these into a single package and get rid of the windows vs linux differences?

Reply:
This is still the plan. There's an issue tracking this and doing this work is getting close to the top of the todo list.

Reply:
This has been addressed.